specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/ContainerApps.json

```diff
@@ -728,6 +731,18 @@
               "$ref": "./CommonDefinitions.json#/definitions/WorkloadProfileName",
               "description": "Workload profile name to pin for container app execution."
             },
+            "s2CAutoPatchConfiguration": {
```

Comment:
Please consider using a generic name and making it an object, so that this property is not tied to Source to Cloud or even Auto Patching.
My suggestion would be to call it "patchingConfiguration", and then have in it the property "patchingMode" with enum values "Disabled", "Manual", "Automatic"

Reply:
Changed in the latest commit.